### ISSUE TITLE
Missing response module

### ISSUE THREAD

**Issue by washcycle** [ahartoto/lendingclub2#13]
I believe the pip package is missing the response folder. I checked the site package an there is no Response folder.

```
(data-sci) washcycle@deathscythe-u:~$ python
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from lendingclub2.account import InvestorAccount
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/washcycle/anaconda3/envs/data-sci/lib/python3.6/site-packages/lendingclub2/account.py", line 17, in <module>
    from lendingclub2.response.notes import Notes
ModuleNotFoundError: No module named 'lendingclub2.response'
```

**Comment by washcycle** [comment]
It doesn't show up in the test because the test is pulling from git and not pypi as far as I can tell.

**Comment by ahartoto** [comment]
Seems that I missed including it in the `setup.py`

**Comment by ahartoto** [comment]
I just released 0.5.1 that fixed it.

**Comment by washcycle** [comment]
Excellent! Thanks for this package too, saved me a ton of work rolling my own.
